src/io/flutter/project/FlutterProjectOpenProcessor.java

```diff
@@ -102,4 +114,33 @@ private ProjectOpenProcessor getDelegateImportProvider(@Nullable VirtualFile fil
   public boolean isStrongProjectInfoHolder() {
     return true;
   }
+
+  private static class PackagesOutOfDateNotification extends Notification {
+
+    @NotNull
+    private final Project myProject;
+
+    public PackagesOutOfDateNotification(@NotNull Project project) {
+      super("Flutter Packages", FlutterIcons.Flutter, "Package updates.",
+            null, "This project's packages are ready to" +
```

Comment:
Let's look at some options together.  @mit-mit: your input would be great here too.

The biggest downside to a more precise message is that if the message is longer than a line, the actionable hyperlink gets hidden beneath the fold.

For example:

<img width="390" alt="screen shot 2017-01-20 at 5 58 38 am" src="https://cloud.githubusercontent.com/assets/67586/22151997/8a920de8-ded5-11e6-8a54-288553adce8b.png">

needs to be expanded to reveal:

<img width="407" alt="screen shot 2017-01-20 at 5 58 43 am" src="https://cloud.githubusercontent.com/assets/67586/22152006/963283da-ded5-11e6-81f2-79d8db193487.png">

An alternative might be to have the action above the fold but provide more details below?  Anyway, yeah, let's definitely refine in another pass.  (There'll be a few more rest assured! 😄 )